src/main/paths.ts

```diff
@@ -20,6 +20,7 @@ const APP_NAME = 'rancher-desktop';
  * DarwinObsoletePaths describes the paths we're migrating from.
  */
 class DarwinObsoletePaths implements Paths {
+  appHome = '*NOT USED*';
```

Comment:
Would probably be better to do like the other ones, and throw on access instead?

Reply:
I thought this would be fine for an introduced field, but I can do that (and I assume for `Win32ObsoletePaths` as well).